src/templates/events/event-template.js

```diff
@@ -116,8 +117,26 @@ export default ({ data, pageContext }) => {
     seo,
   } = frontmatter;
 
+  const todaysDate = new Date();
 
+  const dateString = `${todaysDate.getFullYear()}-${
+    todaysDate.getMonth() + 1 < 10 ? "0" : ""
+  }${todaysDate.getMonth() + 1}-${
+    todaysDate.getDate() < 10 ? "0" : ""
+  }${todaysDate.getDate()}`;
 
+  const parsedEventDate = new Date(Date.parse(date))
+
+  const eventDate = `${parsedEventDate.getFullYear()}-${
+    parsedEventDate.getMonth() + 1 < 10 ? "0" : ""
+  }${parsedEventDate.getMonth() + 1}-${
+    parsedEventDate.getDate() < 10 ? "0" : ""
+  }${parsedEventDate.getDate()}`
+
+  const past = dateString > eventDate ? true : false
```

Comment:
the expression `dateString > eventData` returns `true` or `false` already, so there's no need to check if that expression is `true` to determine whether to return `true` or `false`; you can simply return the expression.